docs/source/concepts/bento.rst

```diff
@@ -591,7 +607,7 @@ validation, specify the desired variant in the under ``python.packages`` field:
            packages:
            - "bentoml[tracing-zipkin]"
 
-   .. tab-item:: OpenTelemetry Protocol
+   .. tab-item:: OTLP
```

Comment:
Why did you change it to the shorter name here?

Reply:
It is actually the name of the protocol, it should be OTLP :)